MAUI/ImageEditor/shape.md

```diff
@@ -51,6 +51,47 @@ The [`AnnotationShape`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Imag
 * DoubleArrow
 * DottedArrow
 * DottedDoubleArrow
+* Polygon
+* Polyline
+
+## Path annotation
```

Comment:
Do not mention as path annotation, add separate topics, polygon and polyline

Reply:
Added separate topics for polygon and polyline